cap/modules/deposit/api.py

```diff
@@ -115,3 +119,28 @@ def clear(self, *args, **kwargs):
     #             raise  # TODO raise REST exception ...
 
     #     return super(CAPDeposit, self).publish(*args, **kwargs)
+
+    @index
+    def clone(self, pid=None, id_=None):
+        """Clone a deposit
+
+        Adds snapshot of the files when deposit is cloned
+        """
+        data = dict(self.dumps())
+        if not data['_deposit']['status'] == 'draft':
+            data['_deposit']['status'] == 'draft'
+        id_ = id_ or uuid.uuid4()
+        deposit_pid = cap_deposit_minter(id_, data)
```

Comment:
We can probably remove the `_deposit` field completely and rely on `Deposit.create`.